test/unit/email_rate_limit_test.rb

```diff
@@ -51,8 +58,10 @@ def seed_array
     day_ago               = rand(last_clicked_days_ago)
     clicked_ago           = valid_values.sample
     bad_clicked_ago        = invalid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
-    assert EmailRateLimit.new(day_ago).skip?(bad_clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).skip?(#{bad_clicked_ago}) to be true, was not"
+    assert EmailRateLimit.new(day_ago).now?(clicked_ago),
```

Comment:
or it can call assert, with a custom error message. your call on what looks better.